test/unit/proxy_rule_test.rb

```diff
@@ -217,7 +217,6 @@ class BackendApiOwned < ActiveSupport::TestCase
       setup do
         service = FactoryBot.create(:simple_service)
         @backend_api = FactoryBot.create(:backend_api, account: service.account)
-        FactoryBot.create(:backend_api_config, backend_api: backend_api, service: service)
```

Comment:
I was looking at this and caught https://github.com/3scale/porta/pull/3781

Reply:
Hehe, good catch!